tests/planner/ompl/test_OMPLPlanner.cpp

```diff
@@ -1,3 +1,4 @@
+#include <LRAstar.hpp>
```

Comment:
Where does this header come from? It should probably in a directory, rather than in the root include path.